test/export/test_export.py

```diff
@@ -661,6 +661,36 @@ def forward(self, x, weight, bias):
                 actual_result.append(node.meta.get("torch_fn"))
         self.assertEqual(actual_result, expected_result)
 
+    def test_export_preserve_linear_at_aot_level(self):
+        class Foo(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(3, 3)
+
+            def forward(self, x):
+                x = self.linear(x)
+                return torch.ops.aten.chunk.default(x, 3, 0)
+
+        gm = torch.export._trace._export(
+            Foo(),
+            (torch.randn(3, 3),),
+            pre_dispatch=False,
+            _preserve_ops=[torch.ops.aten.linear.default],
```

Comment:
+1, it would be nice to have a test that properly errors if you try to include any of:

(1) an op that is not CompositeImplicit
(2) a CompositeImplicit op that is "maybe-aliasing"
(3) a CompositeImplicit op that is mutating (since we will always decompose these in FunctionalTensorMode today)